lecture/src/main/java/de/tum/in/www1/artemis/lecture/service/LectureUnitService.java

```diff
@@ -0,0 +1,67 @@
+package de.tum.in.www1.artemis.lecture.service;
+
+import de.tum.in.www1.artemis.domain.LearningGoal;
+import de.tum.in.www1.artemis.domain.Lecture;
+import de.tum.in.www1.artemis.domain.lecture.LectureUnit;
+import de.tum.in.www1.artemis.repository.LearningGoalRepository;
+import de.tum.in.www1.artemis.repository.LectureRepository;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.*;
+
+@Service
+public class LectureUnitService {
+
+    private final LectureRepository lectureRepository;
+
+    private final LearningGoalRepository learningGoalRepository;
+
+    public LectureUnitService(LectureRepository lectureRepository, LearningGoalRepository learningGoalRepository) {
+        this.lectureRepository = lectureRepository;
+        this.learningGoalRepository = learningGoalRepository;
+    }
+
+    /**
+     * Deletes a lecture unit correctly in the database
+     *
+     * @param lectureUnit lecture unit to delete
+     */
+    @Transactional
```

Comment:
You wrote a comment "// ok because of delete" in the "other" service.
Does this mean, this one is not okay as it doesn't get deleted? Please clarify

Reply:
I removed it as I think it can be omitted. The method's javadoc already mentions that it deletes lecture unit

Reply:
Oooh. It refers to the functionality :D
I thought the comment referred to you moving the files and thus deleting it somewhere 🙈